source/constructs/api/data_source/service.py

```diff
@@ -1763,11 +1776,59 @@ def test_jdbc_conn(jdbc_conn_param: JDBCInstanceSourceBase):
     if not source:
         raise BizException(MessageEnum.SOURCE_JDBC_CONNECTION_NOT_EXIST.get_code(),
                            MessageEnum.SOURCE_JDBC_CONNECTION_NOT_EXIST.get_msg())
-    ConnectionProperties = __glue(account_id, region).get_connection(Name=source.glue_connection)['Connection']['ConnectionProperties']
-    jdbc_url = ConnectionProperties['JDBC_CONNECTION_URL']
-    username = ConnectionProperties['USERNAME']
-    password = ConnectionProperties['PASSWORD']
-    pass
+    glue_database_name = f"TEMP-{jdbc_conn_param.instance_id}"
+    crawler_name = f"TEMP-{jdbc_conn_param.instance_id}"
+    # ConnectionProperties = __glue(account_id, region).get_connection(Name=source.glue_connection)['Connection']['ConnectionProperties']
+    # jdbc_url = ConnectionProperties['JDBC_CONNECTION_URL']
+    # username = ConnectionProperties['USERNAME']
+    # password = ConnectionProperties['PASSWORD']
+    # 根据connection 创建临时glue database
+    response = __glue(account_id, region).create_database(DatabaseInput={'Name': glue_database_name})
+    grant_lakeformation_permission(lakeformation_client, crawler_role_arn, glue_database_name, GRANT_PERMISSIONS_RETRIES)
+    schema = get_schema_from_url(source.jdbc_connection_url)
+    jdbc_targets.append({'ConnectionName': source.glue_connection,
+                         'Path': f"{schema}/%"})
+    # 根据connection 创建临时crawler
+    try:
+        response = __glue(account_id, region).create_crawler(
+            Name=crawler_name,
+            Role=crawler_role_arn,
+            DatabaseName=glue_database_name,
+            Targets={'JdbcTargets': jdbc_targets},
+            TablePrefix='Temp_'
+        )
+        logger.info(response)
+        start_response = __glue(account_id, region).start_crawler(
+            Name=crawler_name
+        )
+        # 创建成功更新字段
+        crud.set_jdbc_instance_connection_status(provider_id=jdbc_conn_param.account_provider_id,
+                                                 account=jdbc_conn_param.account_id,
+                                                 region=jdbc_conn_param.region,
+                                                 instance_id=jdbc_conn_param.instance_id,
+                                                 status="CONNECTED")
+        created = 1
+    except Exception as e:
+        logger.error(traceback.format_exc())
+        crud.set_jdbc_instance_connection_status(provider_id=jdbc_conn_param.account_provider_id,
+                                                 account=jdbc_conn_param.account_id,
+                                                 region=jdbc_conn_param.region,
+                                                 instance_id=jdbc_conn_param.instance_id,
+                                                 status="UNCONNECTED")
+
+    # 删除临时crawler
+    if created == 1:
+        try:
+            __glue(account_id, region).delete_crawler(crawler_name)
+        except Exception:
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears that you are generically passing an `Exception` object without performing any other operation on it. This may hide error conditions that can otherwise be quickly detected and addressed. We recommend that you catch a more specific exception. If the code must broadly catch all exceptions, consider logging the stack trace using the [logging.exception()](https://docs.python.org/3/library/logging.html#logging.exception) API. For example,

```
try:
    x = 1 / 0
except ZeroDivisionError as e:
    logging.exception(&#39;ZeroDivisionError: %s&#39;, e)

```

Similar issue at line number 1827.